pkg/chunk/encoding/delta.go

```diff
@@ -76,21 +76,21 @@ func newDeltaEncodedChunk(tb, vb deltaBytes, isInt bool, length int) *deltaEncod
 }
 
 // Add implements chunk.
-func (c deltaEncodedChunk) Add(s model.SamplePair) ([]Chunk, error) {
+func (c *deltaEncodedChunk) Add(s model.SamplePair) (Chunk, error) {
```

Comment:
Alternative suggestion: delete this whole file.
It's a really old encoding from Prometheus, which nobody has a reason to use.

Reply:
Done :)